app/src/main/java/org/liberty/android/fantastischmemo/ui/MCStudyActivity.java

```diff
@@ -0,0 +1,173 @@
+        rb3 = (RadioButton)findViewById(R.id.mc_radio_button3);
+        rb4 = (RadioButton)findViewById(R.id.mc_radio_button4);
+        buttonConfirmNext = (Button)findViewById(R.id.mc_button_confirm_next);
+
+        textColorDefaultRb = rb1.getTextColors();
+
+        dbOpenHelper = AnyMemoDBOpenHelperManager.getHelper(getApplicationContext(), dbPath);
+        multipleChoiceCardDao = dbOpenHelper.getMultipleChoiceDao();
+        multipleChoiceCardDao.setHelper(dbOpenHelper);
+        multipleChoiceCardList = multipleChoiceCardDao.getAllMultipleChoiceCards();
+
+        mcCardCountTotal = multipleChoiceCardList.size();
+        Collections.shuffle(multipleChoiceCardList);
+
+        showNextMultipleChoiceCard();
+
+        buttonConfirmNext.setOnClickListener(new View.OnClickListener(){
+            @Override
+            public void onClick(View v) {
+                if(!answered) {
+                    if(rb1.isChecked() || rb2.isChecked() || rb3.isChecked() || rb4.isChecked()) {
+                        checkAnswer();
+                    } else {
+                        Toast.makeText(MCStudyActivity.this, "Please select an answer.", Toast.LENGTH_SHORT).show();
+                    }
+                } else {
+                    showNextMultipleChoiceCard();
+                }
+            }
+        });
+    }
+
+    private void showNextMultipleChoiceCard() {
+        rb1.setTextColor(textColorDefaultRb);
+        rb2.setTextColor(textColorDefaultRb);
+        rb3.setTextColor(textColorDefaultRb);
+        rb4.setTextColor(textColorDefaultRb);
+        rbGroup.clearCheck();
+
+        if(mcCardCounter < mcCardCountTotal) {
+            currentMCCard = multipleChoiceCardList.get(mcCardCounter);
+
+            textViewQuestion.setText(currentMCCard.getQuestion());
+            rb1.setText(currentMCCard.getOption1());
+            rb2.setText(currentMCCard.getOption2());
+            rb3.setText(currentMCCard.getOption3());
+            rb4.setText(currentMCCard.getOption4());
+
+            mcCardCounter++;
+            textViewId.setText("ID: " + mcCardCounter + " Total: " + mcCardCountTotal);
+            answered = false;
+            buttonConfirmNext.setText("Confirm");
+        } else {
+            finishMultipleChoice();
+        }
+    }
+
+    private void checkAnswer() {
+        answered = true;
+
+        RadioButton rbSelected = (RadioButton)findViewById(rbGroup.getCheckedRadioButtonId());
+        int answerNr = rbGroup.indexOfChild(rbSelected) + 1;
+
+        switch(answerNr) {
+            case 1: answerSelected = currentMCCard.getOption1();
+            case 2: answerSelected = currentMCCard.getOption2();
+            case 3: answerSelected = currentMCCard.getOption3();
+            case 4: answerSelected = currentMCCard.getOption4();
+                break;
+            default: break;
+        }
+
+/*        if(answerSelected.equals(currentMCCard.getAnswer())) {
```

Comment:
porque?

Reply:
@jameelbrochu  lol

Reply:
It's for a score functionality that I implemented on my branch that will be merged with this PR.